frontend/src/styles/Button.module.scss

```diff
@@ -0,0 +1,23 @@
+@import 'colors';
+
+.landingpagebutton {
+    background: #FFFBFB;
+    border-radius: 17px;
+
+    position: relative;
+    width: 318px;
+    height: 100px;
+    //width: 17%;
+    //height: 10%;
+    margin: 50px;
```

Comment:
Change to rem please

Reply:
Button.module.scss is no longer being used.  Should I delete the file instead?